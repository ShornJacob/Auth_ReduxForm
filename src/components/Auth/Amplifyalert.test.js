import React from 'react';
import AmplifyAlert from './AuthResult';
import renderer from 'react-test-renderer';


it('renders correctly', () => {

 const code = "ACode"
 const message = "AMessage"


  const tree = renderer
    .create(<AmplifyAlert code={code} message={message}></AmplifyAlert>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});