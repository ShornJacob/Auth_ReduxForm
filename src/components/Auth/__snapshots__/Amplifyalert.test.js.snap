// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="fade alert alert-danger show"
  role="alert"
>
   
  AMessage
   
  
</div>
`;
